src/IISIntegration/version.props

```diff
@@ -3,17 +3,15 @@
     <DotNetMajorVersion>2</DotNetMajorVersion>
     <DotNetMinorVersion>2</DotNetMinorVersion>
```

Comment:
What needs to change to make this use the global version.props file?

Reply:
A change similar to this: https://github.com/aspnet/AspNetCore/pull/4013

This can't happen yet because we don't have all of IISIntegration's dependencies merged into the mondo repo.